### ISSUE TITLE
Regression regarding fHighLoad

### ISSUE THREAD

**Issue by Ringdingcoder** [Baron-von-Riedesel/HX#13]
The commented-out `jc exit` in `DPMILDR.ASM`:7339 causes my testcase of running the WinWord 1 OS/2 compiler to crash (testing under WinXP). It was changed in a huge cleanup commit (a0afeeff0c). Reverting just this one line repairs it. It also looks wrong somehow, assuming c is used to indicate failure.

And btw., the missing history and huge commits make it very annoying to pin down a specific change as the guilty one. :disappointed:

**Comment by Baron-von-Riedesel** [comment]
> And btw., the missing history and huge commits make it very annoying to pin down a specific change as the guilty one. disappointed

I agree - the dpmildr.asm source is the oldest part of hx and almost unmaintainable these days - originally written with masm 5.1. Perhaps I should separate this loader to its own repository.

> Reverting just this one line repairs it. It also looks wrong somehow, assuming c is used to indicate failure.
Perhaps, but at this location the loader has been moved to extended memory already. It's no big issue to activate the line, since with hdpmi the call should never fail, though.
